libmpv/cocoa-rendergl/cocoa-rendergl.m

```diff
@@ -138,8 +137,8 @@ @implementation AppDelegate
 
 - (void)createWindow {
 
-    int mask = NSTitledWindowMask|NSClosableWindowMask|
-               NSMiniaturizableWindowMask|NSResizableWindowMask;
+    int mask = NSWindowStyleMaskTitled|NSWindowStyleMaskClosable|
+               NSWindowStyleMaskMiniaturizable|NSWindowStyleMaskResizable;
```

Comment:
fixing the deprecation warnings for the style mask seems like the only related change to this commit. all other changes seem like questionable misc changes(?)

Reply:
Yes for building with clang cli in terminal. And more warnings occurred while building in xCode 9.4.1 with a newly created project.